helpers/deploy/deploymentUtils.ts

```diff
@@ -1,21 +1,33 @@
-import bscMainnetDeployments from "@venusprotocol/venus-protocol/networks/mainnet.json";
-import bscTestnetDeployments from "@venusprotocol/venus-protocol/networks/testnet.json";
-import { getNamedAccounts } from "hardhat";
+import bscMainnetDeployments from "@venusprotocol/venus-protocol/deployments/bscmainnet.json";
+import bscTestnetDeployments from "@venusprotocol/venus-protocol/deployments/bsctestnet.json";
+import { ethers, getNamedAccounts } from "hardhat";
 import { HardhatRuntimeEnvironment } from "hardhat/types";
 
 import { SUPPORTED_NETWORKS } from "./constants";
 
-export const getAcmAdminAccount = async (network: SUPPORTED_NETWORKS): string => {
+export const getAcmAdminAccount = async (network: SUPPORTED_NETWORKS): Promise<string> => {
   const { deployer } = await getNamedAccounts();
   return {
-    bsctestnet: bscTestnetDeployments.Contracts.Timelock, // NORMAL TIMELOCK
-    bscmainnet: bscMainnetDeployments.Contracts.Timelock, // NORMAL TIMELOCK
+    bsctestnet: bscTestnetDeployments.contracts.NormalTimelock.address, // NORMAL TIMELOCK
+    bscmainnet: bscMainnetDeployments.contracts.NormalTimelock.address, // NORMAL TIMELOCK
     sepolia: "0x94fa6078b6b8a26f0b6edffbe6501b22a10470fb", // SEPOLIA MULTISIG
     ethereum: "0x285960C5B22fD66A736C7136967A3eB15e93CC67", // ETHEREUM MULTISIG
     hardhat: deployer,
   }[network];
 };
 
+export const getOmnichainProposalSender = async (network: Omit<SUPPORTED_NETWORKS, "ethereum" | "sepolia">) => {
+  if (network === "hardhat") {
+    const omnichainProposalSenderAddress = (await ethers.getContract("OmnichainProposalSender")).address;
```

Comment:
We should support hardhat for all deployment scripts so that 
1) they can be consumed for integration tests either by the subgraph or other projects
2) We can run the deployment scripts for sanity checking

Often times when I try to run the deployment scripts with hardhat I discover other errors that would prevent them from running on a live network